docker/run.sh

```diff
@@ -54,6 +54,7 @@ function run_tests() {
     now=`date +%s`
     su cchq -c "../run_tests $TEST $(printf " %q" "$@")"
     [ "$TEST" == "python-sharded-and-javascript" ] && scripts/test-make-requirements.sh
+    [ "$TEST" == "python-sharded-and-javascript" && "$TRAVIS_EVENT_TYPE" == "cron" ] && scripts/report-code-metrics.sh datadog
```

Comment:
Any reason to put this here and not have it with the management command - it would make all the metrics appear in the same place in the log output.

Also it would be nice to highlight them a bit so they are easier to see - maybe just a header / footer?

Reply:
good ideas - for some reason I had it in my head that the context of the management command (being saved to a file and run as `cchq`) would complicate things, but on reflection that doesn't really make sense.